src/components/filterPanel/FilterPanel.js

```diff
@@ -240,7 +251,22 @@ const FilterPanel = ({
   }
 
   return (
-    <div className={styles.FilterPanel}>
+    <div
+      className={classNames(
+        styles.FilterPanel,
+        showPanel ? '' : styles.FilterPanelCollapsed
```

Comment:
the classNames helper is used for exactly this situation right? Where you optionally add a class. No ternary needed..?

Reply:
This is a habit I have, but you are totally right. I changed it.